command/init.go

```diff
@@ -168,6 +170,62 @@ job "example" {
         }
       }
 
+      # The template block instantiates an instance of consul-template. This
+      # creates a convenient way to ship configuration files that are populated
+      # from Consul data, retrieve secrets from Vault and even ship static
+      # configurations to Nomad tasks.
+      # template {
```

Comment:
Yes - we will cover this more in the online documentation. This is just the sample job file. Unfortunately we don't want to overload it with too much information, but want to get the general point across.